layer/layer_data.cc

```diff
@@ -25,6 +25,11 @@
 #include "layer_utils.h"
 
 namespace {
+// A function alias used to get the current time_point from chrono and replaces
+// absl::now().
+auto Now = std::chrono::system_clock::now;
+
+using time_point = std::chrono::system_clock::time_point;
```

Comment:
If all layers are supposed to use the same clock and `Now` function, we should move it to layer_utils.